<launch>
    <arg name="namespace" default="uwb_simulation_tag_to_tag_node" />
    <arg name="src_name" default="x500_0" />
    <arg name="dst_name" default="x500_1" />

    <node pkg="uwb_simulation"
        exec="uwb_simulation_tag_to_tag_node"
        namespace="$(var namespace)"
        output="screen">
        <param name="src_name" value="$(var src_name)" />
        <param name="dst_name" value="$(var dst_name)" />
    </node>
</launch>